perception/map_based_prediction/include/map_based_prediction/map_based_prediction_node.hpp

```diff
@@ -106,6 +106,13 @@ struct PredictedRefPath
   Maneuver maneuver;
 };
 
+struct PredictionTimeHorizon
+{
+  double vehicle;
```

Comment:
How about add some comment in this description?
We use "vehicle" prediction in motorcycle and "pedestrian" prediction for bicycle for now and this is not obvious to users.